astro/src/components/get-started/DelegateStep3.astro

```diff
@@ -0,0 +1,20 @@
+---
+import Code from 'astro/components/Code.astro';
+
+---
+<dd>
+In your shell, type the following to start up the application:
+
+<Code code="cd app; npm install; npm run dev" />
+</dd>
+
+Point your browser to the application:
+
+<p>GitPod:
+<Code code="gp preview `gp url 8080`" />
+</p>
+<p>
+Local:
+<Code code="open http://localhost:8080" /></p>
+
+Now, login to the application in your browser using admin@example.com and 'password' to see the integration working.
```

Comment:
when I try to log in, I get the 'account not registered'. Should we have the user log out first after inspecting the admin UI application config? it says to login with admin@example.com but that user is not registered for the application. 

we should do richard@example.com 

I think in a previous version we actually registered admin@example.com for the 'start here app' but we dropped that.

Reply:
Switched to richard@example.com for the login